### ISSUE TITLE
Native: Libraries in the PPX can sometimes conflict with user-code

### ISSUE THREAD

**Issue by lessp** [reasonml-community/graphql-ppx#181]
e.g. if you open another library which also has a module which the ppx uses but doesn't implement the same functions you'll hit `unbound value`. In my case I used another standard library which did not implement `List.assoc`.

```
open AnotherStdlib;

[%graphql {| ... |}]
```

**Comment by jfrolich** [comment]
Yeah I think shipping a runtime module is the best option to keep compatibility with < `4.08`

**Comment by jfrolich** [comment]
We can backfit this on the native ppx now. But the plan is to make the Bucklescript ppx universal so it should definitely be part of that PR.
